src/gluonts/model/rotbaum/_predictor.py

```diff
@@ -122,6 +124,8 @@ def __init__(
         use_feat_static_cat: bool = False,
         use_feat_dynamic_real: bool = False,
         use_feat_dynamic_cat: bool = False,
+        static_cardinality: Optional[List] = None,
```

Comment:
I think we can use an empty list the default. This should simplify the code a bit.

Also, shouldn't this be `static_cardinalities`, since each element is a cardinality?

Reply:
Yup, absolutely right about the cardinalities. I'll make that change. However, for other models, we use a boolean for the use_feat_* hyperparameters. I do agree that it might simplify the code a bit but did not want to do something that might confuse a user if they are using multiple Gluon models.

Reply:
However, if you still think it would be alright to use a list here, I would be happy to make that change

Reply:
> Yup, absolutely right about the cardinalities. I'll make that change. However, for other models, we use a boolean for the use_feat_* hyperparameters. I do agree that it might simplify the code a bit but did not want to do something that might confuse a user if they are using multiple Gluon models.

I think it's fine to do things differently than other algorithms.

The reason why I prefer an empty list is that both `None` and `[]` encode the same thing: the absence of features. However, if we use `None`, we need to explicitly check for that, whilst the empty list just works in some cases.